lib/features/docker_exec_websocket_server.js

```diff
@@ -0,0 +1,80 @@
+const MAX_RANDOM_PORT_ATTEMPTS = 20;
+
+var debug = require('debug')('docker-worker:features:docker_exec_websocket_server.js');
+var DockerExecServer = require('docker-exec-websocket-server').DockerExecServer;
+// var waitForPort = require('../wait_for_port');
+var http = require('http');
+var slugid = require('slugid');
+var Promise = require('promise');
+var url = require('url');
+
+export default class WebsocketServer {
+  constructor () {
+    this.path = '/' + slugid.v4();
+    // this.path = '/a';
+  }
+
+  async started (task) {
+    debug('creating ws server');
+    //TODO: add https support
+    var httpServ = http.createServer();
```

Comment:
Just do:
```js
var httpServ = https.createServer({
  key: fs.readFileSync(task.runtime.ssl.key),
  cert: fs.readFileSync(task.runtime.ssl.certificate)
});
```
And we should have HTTPS.

Note, we might want to read the files asynchronously later... using something like:
```js
var readFile = Promise.denodeify(fs.readFile);
var [key, cert] = await Promise.all([
  readFile(task.runtime.ssl.key),
  readFile(task.runtime.ssl.certificate)
]);
var httpServ = https.createServer({key, cert});
```
I live the unpacking and packing in ES6 so nice :)